<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWin</class>
 <widget class="QMainWindow" name="PlotWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPlot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout1">
      <item>
       <widget class="PlotArea" name="pa" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>647</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actExportImg"/>
    <addaction name="actExportSvg"/>
    <addaction name="actExportPdf"/>
    <addaction name="actExportPrn"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actZoom"/>
    <addaction name="actResetZoom"/>
    <addaction name="actLegend"/>
    <addaction name="actGrid"/>
    <addaction name="actAntialiased"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAbout"/>
    <addaction name="actAboutQt"/>
    <addaction name="actDemo"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actExportImg">
   <property name="text">
    <string>Export to Image</string>
   </property>
  </action>
  <action name="actExportSvg">
   <property name="text">
    <string>Export to SVG</string>
   </property>
  </action>
  <action name="actExportPdf">
   <property name="text">
    <string>Export to PDF</string>
   </property>
  </action>
  <action name="actExportPrn">
   <property name="text">
    <string>Print (Export to PDF/PS)</string>
   </property>
  </action>
  <action name="actZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Legend</string>
   </property>
  </action>
  <action name="actResetZoom">
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actAntialiased">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Antialiased</string>
   </property>
  </action>
  <action name="actDemo">
   <property name="text">
    <string>Demo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotArea</class>
   <extends>QWidget</extends>
   <header>plot_area.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
